hoomd/md/pytest/test_rigid.py

```diff
@@ -136,6 +136,45 @@ def test_create_bodies(simulation_factory, two_particle_snapshot_factory,
     if snapshot.communicator.rank == 0:
         check_bodies(snapshot, valid_body_definition)
 
+    sim.operations.integrator = hoomd.md.Integrator(dt=0.005, rigid=rigid)
+    # Ensure validate bodies passes
+    sim.run(0)
+
+    # Second test with more general testing
+    # detach rigid
+    sim.operations.integrator.rigid = None
+
+    initial_snapshot = lattice_snapshot_factory(n=10)
+    if initial_snapshot.communicator.rank == 0:
+        initial_snapshot.particles.types = ["C", "A", "B"]
+        # Grab the middle particles and a random one to ensure that particle
+        # type ordering with respect to particle tag does not matter for
+        # create_bodies.
+        initial_snapshot.particles.typeid[100:800] = 1
+        initial_snapshot.particles.typeid[55] = 1
```

Comment:
Could we choose this index (55) and range (100-800) to come from `np.random.randint`? That way each time the test was run we would get a different random index and range.

Reply:
`conftest.py` sets the numpy random seed so that tests run with reproducible random numbers. Executing tests with different random seeds each time makes failures very challenging to debug.

Reply:
I guess this approach would only work if test output tells us which random numbers were chosen if the test fails.